karabo_data/geometry2/__init__.py

```diff
@@ -341,7 +341,14 @@ def position_all_modules(self, data, out=None):
         """Deprecated alias for :meth:`position_modules_fast`"""
         return self.position_modules_fast(data, out=out)
 
-    def plot_data_fast(self, data, axis_units='px', frontview=True):
+    def plot_data_fast(self,
+                       data,
+                       axis_units='px',
+                       frontview=True,
+                       ax=None,
+                       figsize=None,
+                       colorbar=True,
+                       **kwargs):
```

Comment:
Actually, 2 minor comments:
- style this has 1 argument per line, the `plot_data` method from the `Snapped` has several on the same line, I would align the style.
- the order of arguments are also in a different order on both method, I would try to have them in the same order unless there is a reason for that.

Reply:
Thanks! Yes, I would like the argument order to be fixed (or make the new arguments keyword-only, then it doesn't matter). I'm relaxed about coding style, but consistency never hurts.

@antarcticrainforest do you still have time to make those changes? If not, one of us will do it tomorrow.

Reply:
Yes I'll do it! But. I sorry I don't find the a different order of arguments. Can someone point me to the right spot?

Reply:
Ok I think I got it now. I think I'll make them keyword arg only